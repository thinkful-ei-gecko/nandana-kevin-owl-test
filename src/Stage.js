import React from 'react';
import './Stage.css';

function Stage(props) {
  return (
    <div></div>
  )
}

export default Stage;