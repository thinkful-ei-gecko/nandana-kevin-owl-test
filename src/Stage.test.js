import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Stage from './Stage';

describe('<Stage />', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Stage />, div);
    ReactDOM.unmountComponentAtNode(div);
  });
});