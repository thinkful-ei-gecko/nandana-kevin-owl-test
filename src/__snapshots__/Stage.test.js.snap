// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Stage /> renders satge as expected 1`] = `
<div
  className="Stage"
>
  <h1>
    On Stage
  </h1>
</div>
`;
